src/room/InCallView.tsx

```diff
@@ -555,5 +564,40 @@ function useParticipantTiles(
     return allGhosts ? [] : tiles;
   }, [matrixRoom, sfuParticipants]);
 
+  // We carry over old tiles from the previous focus for some time after a focus switch
+  // so that the video tiles don't all disappear and reappear.
+  const [isSwitchingFocus, setIsSwitchingFocus] = useState(false);
+
+  useEffect(() => {
+    if (connState === ECAddonConnectionState.ECSwitchingFocus) {
+      setIsSwitchingFocus(true);
+    } else if (isSwitchingFocus) {
+      setTimeout(() => {
+        setIsSwitchingFocus(false);
+      }, POST_FOCUS_PARTICIPANT_UPDATE_DELAY_MS);
+    }
+  }, [connState, setIsSwitchingFocus, isSwitchingFocus]);
+
+  if (
+    connState === ECAddonConnectionState.ECSwitchingFocus ||
+    isSwitchingFocus
+  ) {
```

Comment:
I am assuming `connState` is only set to the `ECSwitchingFocus` case for a short time and `isSwitchingFocus` is used to extend the "show prev. tiles" phase for a longer time period?
If so, I am wondering if we ever hit the case where `connState === ECAddonConnectionState.ECSwitchingFocus` is `true` but `isSwitchingFocus` is false?

Reply:
what happens, if the timeout `POST_FOCUS_PARTICIPANT_UPDATE_DELAY_MS` has passed but there are still missing reconnects?

Reply:
Added a comment in a0226156 which hopefully answers both questions!